common/src/main/java/com/revenuecat/purchases/common/logUtils.kt

```diff
@@ -35,7 +35,7 @@ fun errorLog(error: PurchasesError) {
         PurchasesErrorCode.UnsupportedError,
         PurchasesErrorCode.EmptySubscriberAttributesError,
         PurchasesErrorCode.CustomerInfoError,
-        PurchasesErrorCode.InvalidSubscriberAttributesError -> log(LogIntent.RC_ERROR, error.message)
+        PurchasesErrorCode.InvalidSubscriberAttributesError -> log(LogIntent.RC_ERROR, error.toString())
```

Comment:
alternatively, we could just update `message` to include the `underlyingErrorMessage`